plugin/impl.go

```diff
@@ -185,6 +217,8 @@ func (p *Plugin) Execute() error {
 	switch {
 	case p.settings.Login.Password != "":
 		logrus.Info("Detected registry credentials")
+	case p.settings.Login.RegistriesYaml != "":
+		logrus.Info("Detected multiple registries credentials")
```

Comment:
```suggestion
		logrus.Info("Detected multiple registry credentials")
```

Just a small nitpick

Reply:
f93d34038d1f5810a9b1d85fd6a7cb3114097fca